docs-source/content/samples/simple/domains/model-in-image/update4.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "Update 4"
+date: 2019-02-23T17:32:31-05:00
+weight: 3
+---
+
+This use case demonstrates dynamically configuring Work Manager Threads Constraints in your running domain without restarting the servers. This use case requires Update1 use case to be run already. 
+
+In the use case, you will:
+
+ - Update the ConfigMap containing WDT model created in Update1 use case with changes to Work Manager Threads Constraint configuration
+ - Update the Domain YAML file to enable online update feature
+ - Update the Domain YAML file to trigger a domain introspection, which applies the new configuration values without restarting servers
+
+Here are the steps:
+
+1. Make sure you have deployed the domain from the [Update 1]({{< relref "/samples/simple/domains/model-in-image/update1.md" >}}) use case.
+
+1. Update the model YAML file from the [Update 1]({{< relref "/samples/simple/domains/model-in-image/update1.md" >}}) use case, with the desired changes to the Minimum and Maximum Threads Constraints.
+
+   Here's an example model configuration that updates the configured count values for the `SampleMinThreads` Minimum Threads Constraint and `SampleMaxThreads` Maximum Threads Constraint:
+
+
+   ```
```

Comment:
@jshum2479 We added the data source in a previous update, but we only want to update the SelfTuning configurations here. Do we need to keep the JDBCSystemResource section in the ConfigMap?

Reply:
I think it's helpful to layer changes on top of each-other, where teh first change was not dynamic - so keeping the JDBC system resource helps IMO.    On a related note, it'd be more clear/realistic if teh JDBC system resource remained in its original yaml file, and the updated configmap was built by adding both the original file and the new file together (instead of combining JDBC and WM tuning into the same file).

Reply:
(I think you can have two "--from-file" arguments on the create cm command line.)

Reply:
Thanks for the suggestion! I have removed the datasource updates from the wm updates, and updated scripts to use two "--from-file" arguments when creating cm for update 4.

Reply:
The merged model at any time is the source of truth (i.e. model in the image and configmap are merged into a single model).  The delta is based on the differences between the current running and the newly merged model and then used for update.  So, always add to, remove, update to the configmap to form a final model that you want.  If you remove everything in the configmap and just add a new wm, then all elements in the model will be removed and ending with just the wm.

Reply:
I think we're all on the same page. Resolving. Feel free to unresolve if you disagree.